packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -2581,6 +2581,9 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin, Resto
     super.dispose();
   }
 
+  /// Delegate triggered when we remove the bottom insets
```

Comment:
The doc comments needs to explain in a little more detail when this fires and why someone should care.